tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetWebApi2Tests.cs

```diff
@@ -193,8 +194,28 @@ public async Task SubmitsTraces(string path, HttpStatusCode statusCode, int expe
             // Append virtual directory to the actual request
             var spans = await GetWebServerSpans(_iisFixture.VirtualApplicationPath + path, _iisFixture.Agent, _iisFixture.HttpPort, statusCode, expectedSpanCount);
 
-            var sanitisedPath = VerifyHelper.SanitisePathsForVerify(path);
+            var aspnetSpans = spans.Where(s => s.Name == "aspnet.request");
+            foreach (var aspnetSpan in aspnetSpans)
+            {
+                var result = aspnetSpan.IsAspNet();
+                Assert.True(result.Success, result.ToString());
```

Comment:
maybe this could be an extension method too on the Result too as we'll want to assert after most of the time?

Reply:
I hadn't considered that! For now I'd like to leave this as-is because I prefer having all of the Assert calls in the test case method instead of being distributed in helpers, but I could see extension methods being a good solution to simplifying our test code.